### PR TITLE
enable logging; read in local_settings.py or QUIZ_SETTINGS

### PR DESCRIPTION
**Pull request by tillmo**
solves #17 as well as #25. For the latter, note that local_settings.py.template contains some mysql connection template. A directory for storing data is not needed here; the location of the database is defined by the database installation.

### CODE REVIEW

File: backend/lqserver/lqserver/local_settings.py.template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+LOGFILE = '/path/to/quiz.log'
```
Comment (jelmd) [inline backend/lqserver/lqserver/local_settings.py.template:1]: What about loglevels - would be nice, they can be adjusted as well. E.g. in production one usually wanna see the error/config messages, only. When getting into trouble info or even debug messages might be useful ...

Comment (tillmo) [reply backend/lqserver/lqserver/local_settings.py.template:1]: good idea, added that

**Comment by tillmo** [comment]
@jelmd please review

File: backend/lqserver/lqserver/local_settings.py.template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    'USER': 'DB_USER',
+    'PASSWORD': 'DB_PASSWORD',
+    'HOST': 'localhost',   # Or an IP Address that your DB is hosted on
+    'PORT': '3306',
+    'OPTIONS': {
+        'init_command': "SET sql_mode='STRICT_TRANS_TABLES'",
+        'charset': 'utf8mb4',
+    }
+} }
+
```
[the remark this answers is not in the document]
Comment (tillmo) [reply backend/lqserver/lqserver/local_settings.py.template:21]: it seems so, see https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces and https://stackoverflow.com/questions/119562/tabs-versus-spaces-in-python-programming

**Comment by tillmo** [comment]
good point, have added SECRET_KEY in the template

**Pull request merged**